client/src/components/qa/Question.jsx

```diff
@@ -26,13 +29,29 @@ function Question({
         });
     }
   };
+
+  const toggleAddAnswer = (e) => {
+    e.preventDefault();
+    setAnsOpen(true);
+  };
+
   return (
-    <div id="Question">
-      <div id="qBody">
-        <span id="qBodyText"><b>Q: </b>{body}</span>
-        <span id="qHelpful">Helpful? <span id="qHelp" onClick={handleQHelpful}><u>Yes</u> </span>
-          ({helpfulness})  | <u>Add Answer</u>
+    <div className="Question">
```

Comment:
From what I know, naming convention for classes and ids for CSS is lowercase letters with dashes between words instead of camelCase. Just letting you know for the future projects, it might be too much work to change that at this point.